packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import ButtonGroupButton from './ButtonGroupButton';
+import './ButtonGroup.scss';
+
+const propTypes = {
+  /**
```

Comment:
I think we can discuss this in war room. It's good to have other teammates' opinions in here. My initial thought is like
1. required props come first(should get most attention), boolean props come together(has the same type and follow the same naming convention), functions come together(has the same type and follow the same naming convention).
2. `defaultProps` should be in the same order in the `propTypes`.